stack/core/src/main/java/org/apache/usergrid/persistence/EntityManagerFactory.java

```diff
@@ -189,6 +188,12 @@ public UUID initializeApplication(
 
     public Health getIndexHealth();
 
+    void rebuildAllIndexes(ProgressObserver progressObserver);
```

Comment:
hi @tnine, I still see the line here - https://github.com/apache/incubator-usergrid/blob/two-dot-o-dev/stack/rest/src/main/java/org/apache/usergrid/rest/IndexResource.java#L88
I'll pull and push again